src/pageLayouts/frc/Competition/Competition.js

```diff
@@ -1,11 +1,348 @@
 import React from 'react';
-// import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import Container from '../../../templates/Container';
+import competition from '../../../components/competition';
 
-const Competition = (props) => {
+const Competition = () => {
   return (
-    <div className='competition'>
-      <p>This is the competition page</p>
-    </div>
+    <Container division="frc" title="Competition" subtitle="This is a banner for the competition page.">
+      <div className="competition">
```

Comment:
@urvi-pisal This is the second instance of the `competition` class. Unless they are meant to hold the same styles (and I assume they are not, since they are being used differently), you will need to rename one. Probably rename the component something like: `competition-event`. Using "component" when naming things is tempting, because that's what we call them, but everything is a component, so it ends up not being helpful. 

Naming things is the hardest part of programming! :)